### ISSUE TITLE
Nunjuks global variables

### ISSUE THREAD

**Issue by mgree** [rwoll/submission#27]
Can we initialize nunjuks with some global variables?

- [x] `admin.email`
- [ ] user info for each request

**Comment by mgree** [comment]
I tried doing the following after initializing nunjucks:

```
    app.use(function (req, res, next) {
      var rawRender = res.render;
      console.log(rawRender);
      var user = req.user;
      
      res.render = function (template, ctx) {
        if (!ctx) {
          ctx = {};
        }

        if (ctx && typeof ctx === 'object' && !ctx.user) {
          ctx.user = user;
        }

        return rawRender(template, ctx);
      };

      next();
    });
```
But I got an error:
```
TypeError: Cannot read property 'req' of undefined
    at render (/Users/mgree/submission/node_modules/express/lib/response.js:939:17)
    at ServerResponse.res.render (/Users/mgree/submission/src/server/index.js:88:16)
    at /Users/mgree/submission/src/server/index.js:150:25
    at Layer.handle_error (/Users/mgree/submission/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/Users/mgree/submission/node_modules/express/lib/router/index.js:310:13)
    at /Users/mgree/submission/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/mgree/submission/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/mgree/submission/node_modules/express/lib/router/index.js:271:10)
    at Layer.handle_error (/Users/mgree/submission/node_modules/express/lib/router/layer.js:67:12)
    at trim_prefix (/Users/mgree/submission/node_modules/express/lib/router/index.js:310:13)
    at /Users/mgree/submission/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/mgree/submission/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/mgree/submission/node_modules/express/lib/router/index.js:271:10)
    at next (/Users/mgree/submission/node_modules/express/lib/router/route.js:121:14)
```
Couldn't figure out what went wrong there without diving into express, which I'm not willing to do just now. Downgrading this to a question.

**Comment by mgree** [comment]
The correct thing to do is to use `res.locals`, which all be passed (overlaid with globals) in to calls to `res.render` automatically.

Fixed in f75929443059f511ce30071d14797a6a0c5e977f.
